en/payment-link/overview.md

```diff
@@ -1,41 +1,41 @@
-
-1. **Order ID** - Unique ID for each Payment Link. You can search for created Payment Links using this ID
-2. **Payment Link ID** - The URL address of the Payment Link, for ease of sharing
-3. **Usage Limit** - The maximum number of times the Payment Link can be used
-4. **Item Name** - The name of the product or service you are selling
-5. **Item Price** - The price of the product or service you are charging
+After clicking the **Create** button, you are redirected to Payment Link page. To create a link, the fields such as Order ID, Payment Link ID, Usage Limit, Item Name, and Item Price are required.
 
 ![Create Payment Link](../../asset/image/paymentlink_create.png)
 
-Additional options are available if (a) you want to send the Payment Link to a specific customer or (b) you want to include an expiration time and date.
+| Label | Field | Description | Remarks |
+| ----- | --------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| 1 | Order ID | It is unique ID for each *Payment link*.| Each time, when you create a *Payment Link*, Order Id gets generated.|
+| 2 | Payment Link ID | It is the URL address of the *Payment link*.| You can share the URL of *Payment Link* with your customers or on any social media platform. |
+| 3 | Usage Limit | It is a maximum number of times the *Payment link* can be used.| If this field is set, then you can use the *Payment Link* for specified number of times.|
+| 4 | Item Name | It is the name of the product or service you are selling.| - |
+| 5 | Item Price | It is the price of the product or service you are charging.| - |
 
-![ptions Payment Link](../../asset/image/paymentlink_options.png)
+Additional options are available if (a) you want to send the *Payment Link* to a specific customer or (b) you want to include an expiration time and date.
 
-If the transaction is directed towards a specific customer, the Payment Link will be sent straight to the specified customer’s email address. Expiration Date is the last day the Payment Link can be used and accessed.
+![Options Payment Link](../../asset/image/paymentlink_options.png)
 
-## Manage Payment Link
-
-After creating the Payment Link, you will be redirected to the Payment Link List page where you can view a list of Payment Links created. You can copy and share any of your links to your customers. You can also see details and delete your links from this page.
+If the transaction is intended for a specific customer, the *Payment Link* is sent straight to the specified customer’s email address. *Expiration Date* is the last day, the *Payment Link* can be used and accessed. On the other hand, there is a field, called *Maximum Usage*. It indicates the number of times, the link can be accessed by customers. If the number exceeds before expiry date, then the link is no longer accessible to the customers.
```

Comment:
Hi Rizda. Thank you for your feedback. I have edited the paragraph and pasted it here. Kindly go through it and please let me know if it is appropriate so that I can make the changes and commit the file.
Edited paragraph:-
If the transaction is intended for a specific customer, the Payment Link is sent straight to the specified customer’s email address. There is a field called Expiration Date that contains the date and time until when the Payment Link is valid. There is also a field called Maximum Usage that indicates the number of times the link can be accessed. 
The Payment Link is valid until either the Maximum Usage number is reached, or the expiration date/time is reached, whichever occurs first.